src/Types.d.ts

```diff
@@ -2,15 +2,14 @@
 type AppInfo = {name?: string} & Record<string, unknown>;
 type BufferedFile = {name: string; type: string; file: {data: Buffer}};
 type HttpClientResponseError = {code: number};
-type HttpHeaderValue = string | number | string[];
 type MethodSpec = {
   method: string;
   methodType: string;
   urlParams: Array<string>;
   path?: string;
   fullPath?: string;
   encode: (data: RequestData) => RequestData;
-  validator: (data: RequestData, headers: RequestHeaders) => void;
+  validator: (data: RequestData, headers: {headers: RequestHeaders}) => void;
```

Comment:
Why do we have this nested `headers` field? Is this something we could refactor?

Reply:
Ah, the parameter should be called options: https://github.com/stripe/stripe-node/blob/3e80143129af5046c42cbec338c9049aa525face/src/resources/EphemeralKeys.js#L12